src/main/cpp/commands/RunPivot.cpp

```diff
@@ -0,0 +1,33 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "commands/RunPivot.h"
+
+RunPivot::RunPivot() {
+  // Use Requires() here to declare subsystem dependencies
+  // eg. Requires(Robot::chassis.get());
+  Requires(HatchIntake::getInstance().get());
+}
+
+// Called just before this Command runs the first time
+void RunPivot::Initialize() {}
+
+// Called repeatedly when this Command is scheduled to run
+void RunPivot::Execute() 
+{
+HatchIntake::getInstance()->SetPivot(1);
```

Comment:
No magic numbers. Create an identifier for `1`. Also, don't use `1`. Please use something much much much smaller.